packages/debugger/src/panels/variables/tree.tsx

```diff
@@ -332,7 +334,7 @@ const VariablesBranch = (props: IVariablesBranchProps): JSX.Element => {
   }, [data]);
 
   return (
-    <ul className="jp-DebuggerVariables-branch">
+    <TreeView className="jp-DebuggerVariables-branch">
```

Comment:
The `TreeView` wrapper should be set in `VariablesBodyTree` at the beginning of this file. Not here because it will nest tree view inside tree view.

So here you should simply use a fragment

```suggestion
    <>
```